### DISCUSSION TITLE
basic-a2dp-vs1053 Example

### DISCUSSION THREAD

**Discussion by RetroCanuck** [pschatzmann/arduino-audio-tools#1095] [Q&A] [answered]
Hello

I'm trying to compile the basic-a2dp-vs1053.ino example from the arduino-audio-tools\examples\sandbox\basic-a2dp-vs1053\ folder.

This looks to be a good foundation for a bluetooth tape deck project I've started working on..

I'm getting an error: no matching function for call to 'writeSamples<int16_t>(audio_tools::VS1053Stream&, int16_t*, int&, int)'

The hardware is an ESP32 Devkit C v4 and adafruit VS1053b.

Am I missing something obvious?

Thanks!

**Comment by pschatzmann** [comment, marked as answer]
I committed a correction for the compile error to go away, but please don't forget that [this sketch might not work](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/sandbox)

If it is working, please let me know...
